src/test/java/com/answerdigital/academy/answerking/service/OrderServiceTest.java

```diff
@@ -240,7 +243,7 @@ void testAddProductToBasketWhenProductIsUnavailableThrowsProductUnavailableExcep
                 .name("King Burger")
                 .description("A burger fit for a king")
                 .price(new BigDecimal("12.99"))
-                .available(false)
```

Comment:
This method needs renaming to `testAddProductToBasketWhenProductIsRetiredThrowsRetiredException`

I would also suggest searching the codebase for any strings matching 'available' and see if there are any other ones we've missed.